tensorflow_addons/metrics/f_scores.py

```diff
@@ -64,16 +67,17 @@ class FBetaScore(tf.keras.metrics.Metric):
             number of true instances in each class.
     """
 
+    @typechecked
     def __init__(
         self,
         num_classes,
         average=None,
         beta=1.0,
         threshold=None,
         name="fbeta_score",
-        dtype=tf.float32,
+        dtype: types.AcceptableDTypes = None,
     ):
-        super().__init__(name=name)
+        super().__init__(name=name, dtype=dtype)
```

Comment:
the `dtype` parameter was unused. I also changed the defaut value to be consistant with keras (keras uses K.floatx() when None is given).